### ISSUE TITLE
Feature request show on bottom

### ISSUE THREAD

**Issue by hawkeyexp** [im85288/service.upnext#10]
An option to switch between showing on top/bottom would be nice - in my case i like it more on the bottom. Next time i will start to adapt the new code with the db requests :-)

**Comment by im85288** [comment]
Yes that was something I thought about too but didn’t include it. I’ll definitely add it though and cool I think you’ll find the code is better organised now :) It works really well with the Netflix addon with the popup happening at the correct timing.
